src/compiler/checker.ts

```diff
@@ -1070,7 +1069,7 @@ import {
     WhileStatement,
     WideningContext,
     WithStatement,
-    YieldExpression,
+    YieldExpression

```

Comment:
```suggestion
    YieldExpression,
```

Well, in any case, probably best to leave it until we can figure out what's happening. This is of course a nit (still looking at the PR)